dlme_airflow/utils/dataframe.py

```diff
@@ -38,4 +38,4 @@ def dataframe_to_file(collection):
     )
     source_df.to_csv(working_csv, index=False)
 
-    return working_csv
+    return {"working_csv": working_csv, "source_df": source_df}
```

Comment:
@edsu and @lwrubel i'm interested in your opinions on whether there's a more idiomatic way to do this, since y'all have more python experience.